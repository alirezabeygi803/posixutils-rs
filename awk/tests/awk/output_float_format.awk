BEGIN {
    print OFMT
    print 123.456
    OFMT = "%.2f"
    print 123.456
    OFMT = "%.0f"
    print 123.456
    OFMT = "%e"
    print 123.456
    OFMT = "%a"
    print 123.456
}