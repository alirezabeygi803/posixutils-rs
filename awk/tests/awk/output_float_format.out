%.6g
123.456
123.46
123
1.234560e+02
0x1.edd2f1a9fbe77p+6
